### ISSUE TITLE
How to check the response codes?

### ISSUE THREAD

**Issue by MissiaL** [kiwicom/schemathesis#319]
Hi! Thanks for such a useful tool!
I would like to understand if response codes are checked with an openapi scheme?

**Comment by Stranger6667** [comment]
Hello! I am glad to hear that you find it useful :)

Yes, response codes could be checked with `schemathesis.runner.status_code_conformance` - currently, it is usable only in CLI:

```
$ schemathesis run -c status_code_conformance http://127.0.0.1:8080/schema.yaml
```

there are some more checks:
- not_a_server_error (default one)
- content_type_conformance (checks if responses have a proper content type defined in the schema)
- response_schema_conformance (checks the whole response schema)

Probably they could be adjusted to in-code usage as well - I will take a look

Please, let me know if I can help somehow

**Comment by MissiaL** [comment]
I run schemathesis in my code(pytest)
It would be great if it was possible to check the response codes in the tests too

I see it like this

```    
kwargs = case.as_requests_kwargs()
response = requests.request(**kwargs)
case.validate(response)
```

**Comment by Stranger6667** [comment]
It seems like a great API to me :) Will plan for the next release, thank you for the suggestion

**Comment by Stranger6667** [comment]
`case.verify_response` is released in Schemathesis 0.20.0 - https://pypi.org/project/schemathesis/0.20.0/

Let me know if it will work for you
